lucene/ivy-versions.properties

```diff
@@ -101,10 +101,10 @@ net.thisptr.version = 0.0.8
 
 /org.apache.ant/ant = 1.8.2
 
-org.apache.calcite.avatica.version = 1.10.0
+org.apache.calcite.avatica.version = 1.13.0
 /org.apache.calcite.avatica/avatica-core = ${org.apache.calcite.avatica.version}
 
-org.apache.calcite.version = 1.13.0
+org.apache.calcite.version = 1.18.0
```

Comment:
Calcite uses Guava and gives ability to use latest version here. Calcite 1.16.0 requires later than Guava 19. `calcite.avatica.version` and `janino.version` are upgraded accordingly.